dietpi/dietpi-software

```diff
@@ -4949,11 +4951,8 @@ _EOF_
 			# - RPi
 			elif (( $G_HW_MODEL < 10 )); then
 
-				# Check current firmware, if "rpi-update" is detected, to allow inform user about possible kernel and headers mismatch
-				which rpi-update &> /dev/null && local old_firmware=$(ls /lib/modules/)
-
 				kernel_packages='raspberrypi-bootloader raspberrypi-kernel libraspberrypi-bin libraspberrypi0 raspberrypi-kernel-headers'
-				apt-mark unhold $kernel_packages
+				G_RUN_CMD apt-mark unhold $kernel_packages
```

Comment:
@Fourdee 
For WireGuard it is not required/contra-productive to hold the packages. The module rebuilds itself, when kernel packages are updated via some APT trigger.
So I think for WireGuard it is good as it was.

Only for manual build modules (this i-sabre-km* sound module), we need to set modules on hold or inform user to rebuild. In that case I think it's fine to simply inform user, since one just needs to reselect the sound card.